<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_ALEGIFReaderType</class>
 <widget class="QDialog" name="W_ALEGIFReaderType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>读取方式</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QComboBox QAbstractItemView::item { min-height: 26px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>选择GIF读取器</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1">
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(150, 150, 150);</string>
        </property>
        <property name="text">
         <string>读取透明GIF不会重影，但是遇到压缩过的GIF时，会出现图片失真情况。
</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>QImageReader读取</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>cximage读取</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(150, 150, 150);</string>
        </property>
        <property name="text">
         <string>读取图片不会失真，但是读取透明GIF时，会出现重影问题。
 </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>W_ALEGIFReaderType</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
